pkgs/build-support/release/default.nix

```diff
@@ -21,6 +21,11 @@ let
 in
 
 rec {
+  binaryTarball = throw "removed (8 Jul 2025)";
+  debBuild = throw "removed (8 Jul 2025)";
+  makeSourceTarball = throw "removed (8 Jul 2025); use releaseTools.sourceTarball";
+  mvnBuild = throw "removed (8 Jul 2025)";
+  rpmBuild = throw "removed (8 Jul 2025)";
```

Comment:
Can you make these more like the removal messages in `pkgs/top-level/aliases.nix`? Right now they’d be pretty confusing without a stack trace. (Thankfully it seems like we don’t recurse into here so we don’t need to guard these with an aliases check.)